core/src/index/unittest/test_ivf.cpp

```diff
@@ -118,12 +133,23 @@ TEST_P(IVFTest, ivf_serialize) {
         reader(ret, bin->size);
     };
 
+    {
+        // null faisss index serialize
+        ASSERT_ANY_THROW(index_->Serialize());
+        knowhere::IVFIndexModel model(nullptr);
+        ASSERT_ANY_THROW(model.Serialize());
+    }
+
     {
         // serialize index-model
         auto model = index_->Train(base_dataset, conf);
         auto binaryset = model->Serialize();
         auto bin = binaryset.GetByName("IVF");
 
+        fiu_enable("FaissBaseIndex.SerializeImpl.throw_exception", 1, nullptr, 0);
+        ASSERT_ANY_THROW(model->Serialize());
+        fiu_disable("FaissBaseIndex.SerializeImpl.throw_exception");
```

Comment:
Maybe using RAII idiom for auto-disable.
{
   auto FiuPoint("xxx", ...);
   ASSERT_ANY_THROW(...);
}

Reply:
This is a good idea, we can use it in later work.